src/react-apps/applications/dashboard/src/services/serviceCard.tsx

```diff
@@ -55,10 +55,14 @@ const styles = {
   },
   fontSize_14: {
     fontSize: '14px',
+  },
+};
+
+class ServiceCardComponent extends React.Component<IServiceCardCompontentProps, IServiceCardComponentState> {
+  constructor(_props: any) {
```

Comment:
No need to have this constructor if you're only calling `super()` on the props.

Reply:
removed